packages/apps/intervention/src/features/index.js

```diff
@@ -17,3 +17,5 @@
  */
 
 export {interventionReducer as intervention_intervention} from './interventionSlice';
```

Comment:
caution to alphabetical order in exports (here, slice and api), error due to rebase I think